libweb3jsonrpc/JsonHelper.cpp

```diff
@@ -130,6 +130,9 @@ Json::Value toJson(dev::eth::Transaction const& _t, std::pair<h256, unsigned> _l
         res["blockHash"] = toJS(_location.first);
         res["transactionIndex"] = toJS(_location.second);
         res["blockNumber"] = toJS(_blockNumber);
+        res["v"] = toJS(u256(_t.signature().v));
```

Comment:
Why do you need the cast? `v` is not 32 bytes, but 1 byte.
`res["v"] = toJS(_t.signature().v);` works somehow for `toJson(dev::eth::Transaction const& _t)`

Reply:
Ah I think it doesn't work well https://github.com/ethereum/cpp-ethereum/issues/4999 Maybe you could fix that too,  returning 62 `0`s here is not cool anyway